common/app-starters-security-common/src/main/java/org/springframework/cloud/stream/app/security/common/AppStarterWebSecurityAutoConfigurationProperties.java

```diff
@@ -18,21 +18,21 @@
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
 /**
- * {@code SecurityCommonAutoConfiguration} properties.
+ * {@code DefaultAppsWebSecurityAutoConfiguration} properties.
  *
  * @author Christian Tzolov
  * @author Artem Bilan
  *
  * @since 2.1
  */
-@ConfigurationProperties("spring.cloud.stream.security")
-public class SecurityCommonAutoConfigurationProperties {
+@ConfigurationProperties("spring.cloud.app-starter-stream.security")
```

Comment:
Let me repeat here, because that `Outdated` is going to be lost:

Huh? That's not consensus and it is is not a starter any more in the end for end user. How can we call http-source-kafka as a starter?

I'm confused... 😢